post_install.sh

```diff
@@ -46,8 +46,11 @@ service postgresql restart
 Echo "Figure out our Network IP"
 #Very Dirty Hack to get the ip for dhcp, the problem is that IOCAGE_PLUGIN_IP doesent work on DCHP clients
 #cat /var/db/dhclient.leases* | grep fixed-address | uniq | cut -d " " -f4 | cut -d ";" -f1 > /root/dhcpip
-netstat -nr | grep lo0 | awk '{print $1}' | uniq | cut -d " " -f4 | cut -d ";" -f1 > /root/dhcpip
+#netstat -nr | grep lo0 | awk '{print $1}' | uniq | cut -d " " -f4 | cut -d ";" -f1 > /root/dhcpip
+netstat -nr | grep lo0 | awk '{print $1}' > /root/dhcpip 
```

Comment:
Doesnt this only runs on install? What if IP changes from DHCP after install?

Reply:
Yes that will be the next step but let's fix first the init issue here.